dist/config/index.js

```diff
@@ -3,59 +3,46 @@
 Object.defineProperty(exports, "__esModule", {
   value: true
```

Comment:
Why have we pushed the dist folder, after explicitly mentioning it in gitignore :octocat:

Reply:
oh, right.. it shouldn't be ignored in this repo.. but it should in all other end user middleware..